### DISCUSSION TITLE
WEB RTC Streaming Support

### DISCUSSION THREAD

**Discussion by vijaynkamble86** [Igalia/meta-webkit#424] [Q&A] [answered]
Hello Team,

I have ported meta-webkit on embedded device. And able to load URLs using cog browser.

But we have requirement to display WEB RTC streaming using cog browser. I would like to know whether cog supports websites / urls which has web rtc streaming contents..?

Build Environment - Yocto Dunfell
HW - Qualcomm SoC

Cheers,
-Vijay Navnath Kamble

**Comment by philn** [comment, marked as answer]
Current status: https://wpewebkit.org/about/faq.html#what%E2%80%99s-the-status-regarding-webrtc%3F

The GstWebRTC backend mentioned there is at early stage of development. Might work on some WebRTC sites, might not.
